cla_public/apps/contact/forms.py

```diff
@@ -174,14 +162,15 @@ def process_selected_time(form_time):
             local = local_tz.localize(naive)
             return local.astimezone(pytz.utc).isoformat()
 
+        safe_to_contact = CONTACT_SAFETY[0][0] if self.contact_type.data == CONTACT_PREFERENCE[1][0] else ""
```

Comment:
This is pretty hard to read. I think I would assign the values from the constants to meaningfully named variables first, to make it much clearer what's being done

Reply:
I think i'll convert them to extended_choices.Choices so that we can do CONTACT_SAFETY.SAFE. There isn't much use of it so updating it wouldn't be a hassle